test/gpu/native/gpuAddNums/gpuAddNums_primitive.compopts

```diff
@@ -3,4 +3,4 @@
 nvccDir=$(which nvcc)
 nvccLib=${nvccDir%/bin/nvcc}/lib64
 
-echo "--ccflags --cuda-gpu-arch=sm_60 --savec=tmp -L$nvccLib -lcuda"
+echo "--ccflags --cuda-gpu-arch=sm_60 --ccflags -DHAS_GPU_LOCALE --savec=tmp -L$nvccLib -lcuda"
```

Comment:
I feel like this `-DHAS_GPU_LOCALE` should already be present as a result of your adding it to `RUNTIME_DEFS`. If you don't add it here does it really end up missing from the target compiler command line?

Another question: does this work with LLVM codegen?

Reply:
The nascent native GPU support actually only works with LLVM codegen.